src/main/plugin/iso19139.ca.HNAP/schematron/schematron-rules-common.sch

```diff
@@ -519,6 +521,11 @@
 
       <sch:assert test="$mapWMSCount &lt;= 2">$loc/strings/MapResourcesWMSNumber</sch:assert>
       <sch:assert test="$mapWMSCount = 0 or $mapWMSCount = 2 or $mapWMSCount &gt; 2">$loc/strings/MapResourcesWMS</sch:assert>
+
+      <sch:let name="noneSpatialType" value="not(string(../../gmd:identificationInfo/gmd:MD_DataIdentification/gmd:spatialRepresentationType))"/>
```

Comment:
Why is this related to the spatial type?  FGP also publishes to Open Data so I would expect that the same issue would exists for spatial.
If FGP has another way to handle this case then maybe we should look into how they handle this?

Reply:
There are many legacy data exists with duplicated url for language purpose. If this validation rule applies to them, how we are going to handle the legacy data? Or we just throw the validation error and let user/editor to correct them?